.github/workflows/main.yaml

```diff
@@ -161,6 +165,13 @@ jobs:
           name: ${{ steps.setup.outputs.artifactName }}
           path: ${{ github.workspace }}/${{ matrix.target }}/${{ steps.setup.outputs.artifactFileName }}
 
+      - name: Publish Debug Symbol Artifacts
+        if: ${{ startsWith(matrix.os, 'macos') && success() && (github.event_name != 'pull_request' || steps.seekingTesters.outputs.found == 'true') }}
```

Comment:
If that's the case, is there a reason we're still checking it here? I think there are some weird edge cases where checking `success()` can produce different outcomes because the implicit `success()` call returns a different result than the explicit `success()` call, but I'm drawing a blank on the exact scenarios right now.

Reply:
Oh right, that's an oversight then.

Reply:
```
if: ${{ startsWith(matrix.os, 'macos') && success() && (github.event_name != 'pull_request' || steps.seekingTesters.outputs.found == 'true') }}
```
vs.
```
if: ${{ success() && startsWith(matrix.os, 'macos') && (github.event_name != 'pull_request' || steps.seekingTesters.outputs.found == 'true') }}
```

My other comment about it not being necessary is that it's applied by default unless you call another status check function. See: https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions:
> A default status check of success() is applied unless you include one of these functions.

If, for some reason, we want to highlight that this step only runs on `success` status, we can leave this here. In that case, I'm used to seeing it as the first conditional, hence my original comment.

This is a newly added line, so I'm not sure what this means:
> this PR will only add the necessary bits to get the dSYM files packaged, but leaves the rest as-is.

Reply:
As discussed off-thread, the intention of the change was to keep the format used overall in the FFmpeg build job, as the job itself will see a full update in https://github.com/obsproject/obs-deps/pull/186.